### PR TITLE
AND #141 Fix login error bug

### LINKED ISSUE #141: Toast for could not create account error pops up on login
Could be because the request is being sent twice for some reason. Look into #22 and see if that helps with things.

### PR DESCRIPTION
**Pull request by kangp3**
Closes #141

### CODE REVIEW

File: app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java [hunk 1]
```diff
@@ -196,7 +197,15 @@ private boolean isValidPhoneForm(String phone_number) {
      * after generating the GCM token.
      */
     private class TokenBroadcastReceiver extends BroadcastReceiver {
+        private HashSet<Integer> received_ids = new HashSet<>();
+
         public void onReceive(Context context, Intent intent) {
+            // Check the unique broadcast ID. If it is a repeat, ignore the broadcast.
+            Integer broadcast_id = intent.getIntExtra(Constants.TOKEN_BROADCAST_ID_EXTRA, 0);
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java]: make the default -1 so we're positive something messed up when we do the check on line 205. `Random.nextInt(n)` is from [0, n)

Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java]: Fixed

**Comment by JuliusAlexanderIV** [comment]
lgtm

**Comment by JuliusAlexanderIV** [comment]
I tried it like 5 times and never got the error so... merge it!

**Pull request merged**